controllers/admin/AdminSearchConfController.php

```diff
@@ -364,7 +364,7 @@ public function __construct()
 
     public function initPageHeaderToolbar()
     {
-        if (empty($this->display)) {
+        if (empty($this->display) || $this->display == 'list') {
```

Comment:
@roncarino I think we can remove this `if` completely 🤔

Reply:
I wanted to avoid to show the "add new alias" button while we were editing or effectively creating a new alias.

Reply:
Makes sense 👍